locales/en-US/experiments.ftl

```diff
@@ -126,6 +142,24 @@ pulseContributors1Title = Firefox UX
 pulseContributors2Title = Firefox Platform Product Intelligence
 pulseContributors3Title = Firefox QA
 pulseEolwarning = The Pulse experiment is ending soon, but the data you provided helped us so much that we want to build the feature in Firefox. We will uninstall this experiment soon.
+sendDescription = Encrypt and send files with a link that automatically expires to ensure your important documents don't stay online forever.
+sendIntroduction = <p>Send lets you upload and encrypt large files (up to 1GB) to share online. When you upload a file, send creates a link to pass along to whoever you want. Each link created by Send will expire after 1 download or 24 hours, and all sent files will be automatically deleted from the Send Server.</p> <p>Unlike other Test Pilot experiments. Send does not require an add-on, and can be used in any modern browser.</p>
+sendMeasurements0 = When you use Send, Mozilla receives an encrypted copy of the file you upload, and basic information about the file, such as filename and file size. Mozilla does not have the ability to access the contents of your encrypted file, and only keeps it for the time or number of downloads indicated.
+sendMeasurements1 = To allow you to see the status of your previously uploaded files, or delete them, basic information about your uploaded files are stored on your local device, such as Send’s identifier for the file, the filename, and the file’s unique download link. This is cleared if you delete your uploaded file or upon visiting Send after the file expires.
+sendMeasurements2 = Anyone you provide with the unique link (including the encryption key) to your encrypted file will be able to download and access that file. You should not provide the link to anyone you do not want to have access to your encrypted file.
```

Comment:
this looks fine to me... i suppose it could be "...to access your..." instead of "...to have access to your...", but i'm inclined to keep it as is since @eleemoz wrote it